src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptStreamingFindUsagesPresenter.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using Microsoft.CodeAnalysis.Editor.Host;
+using Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
+{
+    [Export(typeof(IVSTypeScriptStreamingFindUsagesPresenter)), Shared]
+    internal sealed class VSTypeScriptStreamingFindUsagesPresenter : IVSTypeScriptStreamingFindUsagesPresenter
```

Comment:
```suggestion
    internal sealed class VSTypeScriptStreamingFindUsagesPresenterAccessor : IVSTypeScriptStreamingFindUsagesPresenterAccessor
```

Reply:
I'm not sure what adding "Accessor" accomplishes.

Reply:
Distinguishes between cases where TypeScript is adding their own StreamingFindUsagesPresenter, and cases where TypeScript is accessing the one provided by Roslyn.